src/index.js

```diff
@@ -1,36 +1,118 @@
 import './style.css';
 
-const tasks = [
-  {
-    description: 'Wake up',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'Pray',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Eat breakfast',
-    completed: false,
-    index: 2,
-  },
-];
-
+let tasks = [];
 const taskWrapper = document.querySelector('.to-dos');
+const newTask = document.querySelector('.new-task');
+const addNewTask = document.querySelector('.submit');
+const clearAll = document.querySelector('.clear-all');
 
-const showList = () => {
-  taskWrapper.innerHTML = '';
+const addToLocalStorage = () => {
+  localStorage.setItem('myTasks', JSON.stringify(tasks));
+};
+
+const getFromLocalStorage = () => {
+  if (localStorage.getItem('myTasks')) {
+    tasks = JSON.parse(localStorage.getItem('myTasks'));
+  }
+  return tasks;
+};
+
+const resetIndex = (tasks) => {
   for (let i = 0; i < tasks.length; i += 1) {
-    taskWrapper.innerHTML += `
-    <form id="form">
-      <input type="checkbox" id="${tasks[i].index}" name="task" value="task">
-      <label for="${tasks[i].index}">${tasks[i].description}</label>
-      <i class="fa fa fa-times"></i><br>
-    </form>
-    `;
+    let indx = i + 1;
+    indx = i;
+    tasks[i].index = indx;
+  }
+};
+
+const editTask = (desc, index) => {
+  tasks[index - 1].description = desc;
+  addToLocalStorage();
+};
+
```

Comment:
- [ ] Currently, if I try editing item with index 3, it ends up editing item with index 2. Basically, it deletes item of `index - 1`. This means if you try to edit the first item with index 0, you get a TypeError saying that the task you're trying to delete doesn't exist as shown below 👇 
![image](https://user-images.githubusercontent.com/46839548/166102442-532384d3-27a0-4f8b-83c9-10e56b95a039.png)

**NB** This error occurs because of the line I highlighted. I hope with this, you can easily point out what the issue is. It' s a minor one 👍 😎